docs/source/federated-types/sharing-types.mdx

```diff
@@ -137,35 +137,9 @@ To make `Position.z` shareable, you can do one of the following:
 
 ## Differing shared fields
 
-Shared fields can differ between subgraphs in specific ways:
-* The return type of a shared field can vary in nullability (`String` / `String!`).
-  * A shared field's return type _can't_ vary in its core type (`Int` vs. `String`) or in whether it returns a list (`Int` vs. `[Int]`).
-* A field can be omitted from a subgraph, _if_ that field is always [resolvable](../federated-types/composition/#rules-of-composition).
+The type of a shared field can vary in nullability (`String` / `String!`), but _can't_ vary in its core type (`Int` vs. `String`) or in whether it returns a list (`Int` vs. `[Int]`). Shareable value types must also have the same fields in _every_ subgraph that defines them.
```

Comment:
It actually looks as though this is covered (with examples) in the following subsections, e.g.: https://www.apollographql.com/docs/federation/federated-types/sharing-types#return-types

(That's not to say these sections couldn't use a little copyedittng too...)

Instead of rehashing it in the intro, I might say something like this:

```suggestion
Shared fields can only differ in their [return types](#return-types) and [arguments](#arguments) in specific ways.
If fields you want to share between subgraphs differ more than is permitted, use  [entities](../entities) instead of shareable value types.
```